.drone.yml

```diff
@@ -0,0 +1,81 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/user_ldap
+
+pipeline:
+  ldap:
+    image: dinkel/openldap
+    detach: true
+    pull: true
+    environment:
+      - SLAPD_DOMAIN=owncloud.com
+      - SLAPD_ORGANIZATION=ownCloud
+      - SLAPD_PASSWORD=admin
+      - SLAPD_ADDITIONAL_MODULES=memberof
+    commands:
+      - /entrypoint.sh
+      - slapadd -F /etc/ldap/slapd.d -l /var/www/owncloud/apps/user_ldap/tests/acceptance/ldap-users.ldif
+      - slapd -d 32768 -u openldap -g openldap
+
+  install-server:
+    image: arturneumann/owncloud-ci-php:latest
+    pull: true
+    commands:
+      - cd /var/www
+      - wget -q http://78.47.62.142/archisdownloads/owncloud-daily-master-qa-patched.tar.bz2
+      - tar -xjf owncloud-daily-master-qa-patched.tar.bz2
+      - cd /var/www/owncloud/
+      - php occ maintenance:install -vvv --database=sqlite --database-name=owncloud --database-table-prefix=oc_ --admin-user=admin --admin-pass=admin --data-dir=/var/www/owncloud/data
+      - cd /var/www/owncloud/apps/user_ldap
+      - make
+      - cd /var/www/owncloud/
+      - php occ a:l
```

Comment:
Everyone seems to use shorthand for ``app:list`` ``app:enable`` ...

Reply:
Regarding install - can you switch to using https://github.com/owncloud-ci/core

It allows for using a branch/tarball